packages/react-components/src/components/Cards/Content/index.tsx

```diff
@@ -28,7 +28,7 @@ export default ({
   item,
   config,
   theme = styles,
-  Container = 'div',
+  Container = 'a',
```

Comment:
@pascalachkar let's remove `Container` prop at all. Also pls provide `href` attribute to the `a` element (it should be equal to `item.get('url')`)

Reply:
Done